### ISSUE TITLE
Police Garage Issue

### ISSUE THREAD

**Issue by jokernights** [JonasDev99/qb-garages#14]
I just downloaded this and love how it works. but i am not sure if i am doing something wrong or how it works. With the other one i already had police vehicles set but no matter where i go i can't pull out the police vehicles, same with ambulance? This is in policejob/config.lua
![image](https://user-images.githubusercontent.com/77025615/165680729-7c0cda12-37bc-4573-8fcf-9885e003246b.png)
![image](https://user-images.githubusercontent.com/77025615/165680822-7caa563a-5ae6-46b8-b72a-d5f2ce4797d2.png)

These are in ambulancejob/config.lua 
![image](https://user-images.githubusercontent.com/77025615/165680912-85d80f7c-fca2-4834-8a9d-fa478a0b9e23.png)
![image](https://user-images.githubusercontent.com/77025615/165680934-a6d3312a-adf3-4eaa-ba98-f0edbcc78e55.png)

but when i do the radial menu and open garage it says no vehicles stored? is there a special way to get emergency vehicles out with proper grade?

**Comment by JonasDev99** [comment]
This garage script allows only owned cars to be stored or retreived. It is not a replacement for the vehicle spawner included with qb-policejob.

**Comment by jokernights** [comment]
> This garage script allows only owned cars to be stored or retreived. It is not a replacement for the vehicle spawner included with qb-policejob.

So is there a way to have the best of both worlds? Or do I have to put the original garages back?

**Comment by JonasDev99** [comment]
For now, I would use the qb-policejob spawner but I am planning on adding this functionality as it is frequently requested.

**Comment by jokernights** [comment]
> For now, I would use the qb-policejob spawner but I am planning on adding this functionality as it is frequently requested.

Is this a seperate script? Because it was working with original garages. So would I look for policjob spawner or does that mean I have to go back to original garages?

**Comment by JonasDev99** [comment]
No, you can keep this garages script, but in qb-policejob there is a location defined where you can get policecars. Use that

**Comment by JonasDev99** [comment]
I am not sure, where did you get that code anyway? it is not from my script

**Comment by jokernights** [comment]
> I am not sure, where did you get that code anyway? it is not from my script

No that is the config in policejob that is where they spawned with the original garages in place. Now when I go to that spot the new garages pulls ups but. Says there is nothing in the garage ..  because I believe the garage and policjob worked hand in hand .. so the cars that are assigned are no longer there with the new garages

**Comment by JonasDev99** [comment]
No, qb-policejob never worked together with qb-policejob. You are probably in the wrong location if the option doesn't show up.

**Comment by jokernights** [comment]
> No, qb-policejob never worked together with qb-garages - they are two different scripts, always have been. You are probably in the wrong location if the option doesn't show up.

Nope I’m in the position that the original spawn location was that I set and when I had original garages it allowed me to pull out police vehicles and ambulances now it doesn’t give me that option

**Comment by JonasDev99** [comment]
Then you probably broke it because its not related to qb-garages,

**Comment by JonasDev99** [comment]
Ok, let me have a look.

**Comment by JonasDev99** [comment]
https://github.com/qbcore-framework/qb-policejob/blob/86930ece3f80d095e34906ec275900808e53eac2/client/job.lua#L149
https://github.com/qbcore-framework/qb-policejob/blob/86930ece3f80d095e34906ec275900808e53eac2/client/job.lua#L191
https://github.com/qbcore-framework/qb-policejob/blob/86930ece3f80d095e34906ec275900808e53eac2/client/job.lua#L205

**Comment by jokernights** [comment]
> Ok, let me have a look.

All I did was remove qb garages and installed yours I didn’t break anything and it didn’t work the way it was originally setup

**Comment by JonasDev99** [comment]
Remove the PD Garage (the one in my garage script) at that location and see if it fixes it.

**Comment by JonasDev99** [comment]
Join my discord so we can continue there https://discord.gg/ZcrxXcqvJj

**Comment by jokernights** [comment]
> Join my discord so we can continue there https://discord.gg/ZcrxXcqvJj

Ok I will when I get home

**Comment by jokernights** [comment]
> Join my discord so we can continue there https://discord.gg/ZcrxXcqvJj

Hey thank you so much. I --[[ commented out all the police and pdgarages and it worked .. so the normal garages work and the PD and ambulance work now! .. you said in the future you may add this option in ?

**Comment by JonasDev99** [comment]
Yes

**Comment by jokernights** [comment]
> Yes
![image](https://user-images.githubusercontent.com/77025615/165859386-e464ae5f-0d38-43f8-b288-969fcc9141e9.png)
another question.. so i am testing house garages? is this not in yet? it says vehicle cannot be stored

**Comment by JonasDev99** [comment]
House garages should work, do you own the car? Otherwise I will have another look, might have broken parking for qb-houses by enabling support for loaf housing

**Comment by JonasDev99** [comment]
Could you make another issue so I can close this one? Just so I see what needs fixing without having to read the convos again.

**Comment by JonasDev99** [comment]
What I meant to say is that I may have broken the script by adding loaf housing but yes, make sure that it's set to false.
